tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2931,13 +2931,21 @@ def rgb_to_yiq(images):
   Outputs a tensor of the same shape as the `images` tensor, containing the YIQ
   value of the pixels.
   The output is only well defined if the value in images are in [0,1].
+  
+  Usage Example:
+    ```python
+    >>> x = tf.constant([[[1.0, 2.0, 3.0]]])
+    >>> y = tf.image.rgb_to_yiq(x)
+    [[[ 1.815      -0.9...  0.09...]]]
```

Comment:
This probably needs to read <tf.Tensor(..., numpy=[[[ 1.815  -0.9... 0.09...]]])>

Reply:
should I provide a print(y.numpy()) ?